helios-api/src/lib.rs

```diff
@@ -25,7 +25,8 @@ use tracing::{
 
 use helios_legacy::{ProxyConfig, ProxyState, proxy};
 use helios_remote::PollRequest;
-use helios_state::models::{App, AppTarget, Device, DeviceTarget};
+use helios_remote::input::AppTarget;
```

Comment:
This feels wrong. I think that probably the Local API app input should be different than the remote type and the type defined in state, so it remains consistent even if these other types changes (e.g. by going to target state endpoint v4). 

I'll just add a comment for now and leave it as is to avoid validating and converting twice